packages/bitmasks/bitmasks.1.0.0/opam

```diff
@@ -15,6 +16,7 @@ remove: [
 build-doc: [ "ocaml" "setup.ml" "-doc" ]
 depends: [
   "ocamlfind"
```

Comment:
Could you add a ```build``` attribute if you're not using the ```Findlib``` library ?

Reply:
Yes - oops!